<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>1001</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="detailTAB">
     <attribute name="title">
      <string>Inventory Details</string>
     </attribute>
     <widget class="QLabel" name="detailInvLbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>381</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>Sereach for reference with count level lower or equal: </string>
      </property>
     </widget>
     <widget class="QTableWidget" name="listTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>911</width>
        <height>301</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>partName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>minArea</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>maxArea</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>minDiameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>maxDiameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NumberOfHoles</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QSpinBox" name="detailInvCountSpinBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="searchBtn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="staticTAB">
     <attribute name="title">
      <string>Inventory Statics</string>
     </attribute>
     <widget class="QLabel" name="refereceLbl">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>Number of Reference:</string>
      </property>
     </widget>
     <widget class="QLabel" name="referenceCounterLbl">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="minNbrOfHolesLbl">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>min Numbers of Holes:</string>
      </property>
     </widget>
     <widget class="QLabel" name="minHolesLbl">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="maxHolesLbl">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>100</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="partsLbl">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>Number of parts type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="partsCounterLbl">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="maxNbrOfHolesLbl">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>max Numbers of Holes:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="topTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>691</width>
        <height>141</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>partName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="checkBtn">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>240</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
     <widget class="QLabel" name="topLbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>212</y>
        <width>471</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>Top 3 Reference with minium Inventory Level:</string>
      </property>
     </widget>
     <widget class="QLabel" name="minHolesReferenceLbl">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="maxHolesReferenceLbl">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="editTAB">
     <attribute name="title">
      <string>Edit Inventory</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>691</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="5,5,5,5">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="referenceLbl">
         <property name="text">
          <string>Reference:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="minAreaLbl">
         <property name="text">
          <string>min Area:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="minDiameterLbl">
         <property name="text">
          <string>min Diameter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="nbrOfHolesLbl">
         <property name="text">
          <string>Number of Holes:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="maxDiameterLbl">
         <property name="text">
          <string>min Diameter:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="partNameLbl">
         <property name="text">
          <string>Part Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="maxAreaLbl">
         <property name="text">
          <string>max Area:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="countLbl">
         <property name="text">
          <string>Count:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QSpinBox" name="editInvCountSpinBox"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="referenceText"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="partNameText"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="maxAreaText"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="minAreaText"/>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="nbrOfHolesText"/>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="minDiameterText"/>
       </item>
       <item row="2" column="5">
        <widget class="QLineEdit" name="maxDiameterText"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="updateBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteBtn">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>340</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="firstBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previouseBtn">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextBtn">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="lastBtn">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="idLbl">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;Rockwell&quot;;</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="idLabel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Rockwell&quot;;
background-color: rgb(154, 154, 154);
color: rgb(71, 213, 213);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>1051</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);
font: 75 16pt &quot;Rockwell&quot;;
background-color: rgb(144, 144, 144);</string>
    </property>
    <property name="text">
     <string>Parts Inventory Manager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshBtn">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>580</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="prozessLabel">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>580</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
